executor/tests/services/test_task_identity.py

```
@@ -0,0 +1,23 @@
+
+"""Tests for task-scoped skill identity context building."""
+
+from executor.services.task_identity import build_task_identity_context
+from shared.models.execution import ExecutionRequest
+
+
+def test_build_task_identity_context_returns_standard_skill_identity_env():
+    """Task identity context should expose the standard skill identity env keys."""
+    request = ExecutionRequest(
+        user_name="alice",
+        skill_identity_token="skill-jwt",
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to argument: "skill_identity_token"

(S106)